### ISSUE TITLE
Incorrect Test Case - 1732. Find the Highest Altitude

### ISSUE THREAD

**Issue by CosmicYogi** [LeetCode-Feedback/LeetCode-Feedback#25438]
### LeetCode Username

mitesh504

### Problem Number, Title, and Link

https://leetcode.com/problems/find-the-highest-altitude

### Bug Category

Incorrect test case *(Output of test case is incorrect as per the problem statement)*

### Bug Description

For this test case [-4,-3,-2,-1,4,3,2] the answer should be -1 rather than 0.

### Language Used for Code

Swift

### Code used for Submit/Run operation

```bash
class Solution {
    func largestAltitude(_ gain: [Int]) -> Int {
        var maxAltitude = Int.min
        var sum = 0
        for g in gain { 
            sum += g
            print(g, sum)
            maxAltitude = max(sum, maxAltitude)
        }
        return maxAltitude
    }
}
```

### Expected behavior

It should have passed the test case.
If I initialise the variable maxAltitude with 0 it will pass the test case but it would be incorrect.
Basically [-4,-3,-2,-1,4,3,2] is the array and in this if we traverse from left to right by adding the elements the maximum we would actually be able to achieve will be -1.
